Havit.Blazor.Components.Web.Bootstrap/Toasts/HxMessenger.razor

```diff
@@ -5,13 +5,12 @@
         @foreach (MessengerMessage message in messages)
         {
             // @key is used to pair messages to HxToasts when some message disappers.
+            // We are not passing the Title and Text properties of the MessengerMessage to the HxToast because we are using them to build the ContentTemplate.
```

Comment:
@Harvey1214 This is not true. It is just the extension-methods which use the `ContentTemplate` approach, but you can still pass the full `MessengerMessage` instance via `IHxMessengerService.AddMessage(MessengerMessage message)` and use the `Title` and `Text` properties of the `HxToast`.